overview/default.nix

```diff
@@ -184,26 +184,56 @@ let
         '');
     };
 
-    projects = {
-      one = name: project: ''
-        <article class="page-width">
-          ${heading 1 name}
-          ${render.metadata.one project.metadata}
-          ${render.packages.many (pick.packages project)}
-          ${render.options.many (pick.options project)}
-          ${render.examples.many (pick.examples project)}
-        </article>
+    # The indivdual page of a project
+    projects.one = name: project: ''
+      <article class="page-width">
+        ${heading 1 name}
+        ${render.metadata.one project.metadata}
+        ${render.packages.many (pick.packages project)}
+        ${render.options.many (pick.options project)}
+        ${render.examples.many (pick.examples project)}
+      </article>
+    '';
+
+    deliverableTags = rec {
+      one = label: ''
+        <span class="deliverable-tag">${label}</span>
       '';
-      # Many projects are rendered as links to their individual project sites
       many =
-        projects:
-        concatLines (
-          mapAttrsToList (name: _: ''
-            <article>
-              <a href="/project/${name}">${name}</a>
-            </article>
-          '') projects
-        );
+        project:
```

Comment:
Just a naming nit, but I believe the input should be plural (`projects`) for the many , right?

Reply:
No, because this is for many tags that belong to a singular project. I decided to extract the list of tags in the `render.deliverableTags` section as this logic is about nothing else.

Reply:
Ah, that makes sense.